internals/cli/cmd_okay.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2023 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/canonical/go-flags"
+
+	"github.com/canonical/pebble/client"
+)
+
+const cmdOkaySummary = "Acknowledge notices and warnings"
+const cmdOkayDescription = `
+The okay command acknowledges notices up to most recent notice listed with
+'pebble notices'. After executing okay, 'pebble notices' will only list the
+notices that have occurred (or been repeated) more recently.
+
+The okay command also acknowledges warnings up to the most recent warning
+listed with 'pebble warnings'.
```

Comment:
It migth be more clear if we integrate this into the earlier sentence. Trying it at once:

> The okay command acknowledges warnings and notices that have been previously visualized via the respective commands so they are omitted from future runs of either command. When a notice or warning is repeated, it will again show up until being acknolwedged.

Reply:
Done.